modules/react/select/stories/examples/Basic.tsx

```diff
@@ -1,21 +1,49 @@
 import React from 'react';
 import {FormField} from '@workday/canvas-kit-react/form-field';
-import {Select, SelectOption} from '@workday/canvas-kit-react/select';
+import {Select} from '@workday/canvas-kit-react/select';
+import {Flex} from '@workday/canvas-kit-react/layout';
+
+const options = [
+  {id: 'E-mail', data: {textValue: 'foo'}},
```

Comment:
This may be a moot point after [simplifying the examples](https://github.com/Workday/canvas-kit/pull/2309#discussion_r1350931690), but we should remove `data: {textValue: 'foo'}` if there are any lingering around after simplifying the examples.

Reply:
yes!

Reply:
fixed